client/extensions/woocommerce/woocommerce-services/views/label-settings/label-settings.js

```diff
@@ -197,14 +208,30 @@ class ShippingLabels extends Component {
 			);
 		};
 
+		const openDialog = () => {
+			this.props.openAddCardDialog( siteId );
+		};
+
 		return (
 			<div>
 				{ this.renderPaymentPermissionNotice() }
 				<p className="label-settings__credit-card-description">{ description }</p>
+
+				<QueryStoredCards />
 				{ paymentMethods.map( renderPaymentMethod ) }
-				<Button href={ getOrigin() + '/me/purchases/add-credit-card' } target="_blank" compact>
+
+				<AddCardDialog siteId={ siteId } />
+				<Button onClick={ openDialog } compact>
 					{ buttonLabel }
 				</Button>
+				<Button
```

Comment:
Please add a comment explaining that this will be shown just in WP-Admin using CSS. If you don't have the context, this looks like dead code or a merge conflist badly resolved.